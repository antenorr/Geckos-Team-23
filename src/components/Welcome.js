import React from 'react';

const Welcome = () => (
  <div id="welcome" className="main">
    <h1>Welcome to OneTab ++!</h1>
    <p>OneTab++ will help you organize your favorite pages into categories for easy access.</p>
    <p>When you visit a web page you'd like to add, right-click and add it to a category.</p>
    <p>You can customize your experience from the Settings menu.</p>
    <p>To get started, create your first category!</p>
  </div>
);

export default Welcome;