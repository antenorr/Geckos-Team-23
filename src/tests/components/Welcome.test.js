import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Welcome from '../../components/Welcome';

test('should render Welcome correctly', () => {
  const wrapper = shallow(<Welcome />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});