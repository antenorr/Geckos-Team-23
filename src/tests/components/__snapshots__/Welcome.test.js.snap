// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Welcome correctly 1`] = `
<div
  className="main"
  id="welcome"
>
  <h1>
    Welcome to OneTab ++!
  </h1>
  <p>
    OneTab++ will help you organize your favorite pages into categories for easy access.
  </p>
  <ul
    className="welcome-list"
  >
    <li>
      <p>
        First you'll need to create a category to group your pages
      </p>
    </li>
    <li>
      <p>
        You can have as many categories as you'd like, but every page must belong to a category
      </p>
    </li>
    <li>
      <p>
        When you visit a web page you'd like to add, right-click and add it to a category
      </p>
    </li>
  </ul>
  <p>
    You can customize your experience from the Settings menu.
  </p>
  <p>
    To get started, create your first category!
  </p>
</div>
`;
